### PR TITLE
Ft jwt auth 160788533

### PR DESCRIPTION
**Pull request by mubstimor**
#### What does this PR ​do​?
Adds JWT authentication to endpoints

#### Task to be completed
- auth/login
- auth/signup

#### How should this be manually tested?
Test token availability with Postman

#### Relevant pivotal tracker stories​
- #160955781
- #160788533

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
just fix your failing tests otherwise your good to merge

**Comment by kafuuma** [comment]
Please improve on your test coverage
write more testcases

File: api/views.py [hunk 1]
```diff
@@ -59,6 +69,43 @@ def index():
...
+        'ok': False,
+        'message': 'Missing Authorization Header'
+    }), 401
+
+
+def admin_token_required(f):
+    @wraps(f)
+    def decorated(*args, **kwargs):
+
+        # data, status = get_logged_in_user(request)
```
Comment (Sebuliba-Adrian) [inline api/views.py]: Keep your code base clean.

**Review by Sebuliba-Adrian — CHANGES_REQUESTED** [review]
@mubstimor   Please clean up your code base before

**Comment by mubstimor** [comment]
> just fix your failing tests otherwise your good to merge

I have worked on it, thanks

**Comment by mubstimor** [comment]
> I see you always use the same json object is some of your tests maybe you can put it in varaible in the set method and call the variable

I fixed that anomaly in a later update, thanks.

File: api/views.py [hunk 2]
```diff
@@ -7,7 +8,13 @@
 from flask import Blueprint, render_template as view, render_template
 from jinja2 import TemplateNotFound
 from flasgger import Swagger
-from flask_jwt_extended import (JWTManager, jwt_required, create_access_token, get_jwt_identity)
+from functools import wraps
+# from flask_jwt import JWT, jwt_required, current_identity
```
[the remark this answers is not in the document]
Comment (mubstimor) [reply api/views.py]: I have fixed this in my most recent PR. Thank you.

File: api/views.py (hunk 1 above)
Comment (mubstimor) [reply api/views.py]: I integrated PEP8 to my IDE, so I will be sure to follow the standards in the coming PRs. Thank you.

**Pull request merged**